docs/resources/rediscloud_active_active_subscription_database.md

```diff
@@ -90,6 +88,7 @@ output "us-east-2-private-endpoints" {
 The following arguments are supported:
 * `subscription_id`: (Required) The ID of the Active-Active subscription to create the database in. **Modifying this attribute will force creation of a new resource.**
 * `name` - (Required) A meaningful name to identify the database. **Modifying this attribute will force creation of a new resource.**
+* `redis_version` - (Optional) The Redis version of the database. If omitted, the Redis version will be the default. You can not update this attribute after creation.
```

Comment:
forcenew is in the schema so you can update it, just it will force destroy the db. I'll modify this line to be in line with the rest of the docs.